examples/pyolite - ipywidgets.ipynb

```diff
@@ -0,0 +1,144 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bab23f8-de91-43c9-9cec-84f4924425fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipywidgets import IntSlider"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a15c5acb-ee72-4005-8761-5693db853f22",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider = IntSlider()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ba89682-e0d7-4bd2-961a-f9956850fd5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50510ade-668f-4477-8cb2-41574609ac73",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7bac1ed8-8c77-426b-a781-1c1a6cfad829",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider.value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "976a70a0-e99d-4c20-b005-f59bbba10f85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider.value = 5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "23b32857-2958-4083-b16a-ac26cd2408d4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bqplot import *\n",
```

Comment:
We'll also need the `micropip.install('bqplot')` for `bqplot` to be available in the kernel.

Reply:
Yes. I was thinking of running `micropip.install('bqplot')` directly in the Notebook, what do you think?

Although we might not want to convey the wrong idea that any widget can be installed in the example notebook...

Reply:
But still, it's better than having pyolite install bqplot automatically.

Reply:
Yes for the example notebook I think it's completely fine.

We could also add a comment saying that bqplot works, but other widgets might not.